doc/lsst.obs.decam/using-obs-decam.rst

```diff
@@ -0,0 +1,124 @@
+
+   When running the ``RunIsrForCrosstalkSources.yaml`` pipeline, append ``-c overscan:overscan.fitType='MEDIAN_PER_ROW'``.
+
+.. prompt:: bash
+
+   butler ingest-raws REPO /path/to/raw/science/*.fits.fz --transfer link
+   butler ingest-raws REPO /path/to/raw/calibs/*.fits.fz --transfer link
+   pipetask run -b REPO -i DECam/raw/all -o DECam/raw/crosstalk-sources -p $CP_PIPE_DIR/pipelines/DarkEnergyCamera/RunIsrForCrosstalkSources.yaml --register-dataset-types
+
+Check that this worked successfully, e.g.,
+
+.. prompt:: bash
+
+   butler query-dimension-records REPO exposure --where "instrument='DECam' AND exposure.observation_type='zero'"
+   butler query-dimension-records REPO exposure --where "instrument='DECam' AND exposure.observation_type='dome flat'"
+
+Use `~lsst.cp.pipe`_ to build nightly calibs
+--------------------------------------------
+
+Next, build nightly (or similar) bias and flat frames and certify them into a calib collection.
+
+This example assumes the user has a single night of observations with bias frames numbered 1-6 and flat frames numbered 7-12, and wants to create nightly calib products valid for 24 hours.
+Note the bias building pipeline is camera-agnostic, but the flat-building pipeline has a prerequisite DECam-specific step which correctly handles inter-chip crosstalk.
+
+.. note::
+   Until `DM-30651 <https://jira.lsstcorp.org/browse/DM-30651>`__ is resolved, consider including the following configurations during bias and flat building.
+
+   When running the ``cpBias.yaml`` pipeline, append ``-c isr:overscan.fitType='MEDIAN_PER_ROW'``.
+
+   When running the ``cpFlat.yaml`` pipeline, append ``-c isr:overscan.fitType='MEDIAN_PER_ROW' -c cpFlatNorm:level='AMP'``.
+
+.. prompt:: bash
+
+   pipetask run -d "exposure IN (1, 2, 3, 4, 5, 6)" -b REPO -i DECam/raw/all,DECam/calib -o u/username/bias-construction-night1 -p $CP_PIPE_DIR/pipelines/cpBias.yaml --register-dataset-types
+   butler certify-calibrations REPO u/username/bias-construction-night1 DECam/calib/run1 bias --begin-date 2021-01-01T00:00:00 --end-date 2021-01-01T23:59:59
+   pipetask run -d "exposure IN (7, 8, 9, 10, 11, 12)" -b REPO -i DECam/raw/all,DECam/raw/crosstalk-sources,DECam/calib -o u/username/flat-construction-night1 -p $CP_PIPE_DIR/pipelines/DarkEnergyCamera/cpFlat.yaml --register-dataset-types
+   butler certify-calibrations REPO u/username/flat-construction-night1 DECam/calib/run1 flat --begin-date 2021-01-01T00:00:00 --end-date 2021-01-01T23:59:59
+
+Science time!
+-------------
+
+Now you can proceed with running ISR and other "processCcd" tasks via a Gen3 pipeline.
+
+.. note::
+   Until `DM-30651 <https://jira.lsstcorp.org/browse/DM-30651>`__ is resolved, consider configuring ``-c isr:overscan.fitType='MEDIAN_PER_ROW'`` as above when running ISR.
+
+Some useful pipelines can be found in the ``pipelines/DarkEnergyCamera`` directory of the `~lsst.ap.pipe`_ package.
```

Comment:
I think this needs to be:

```
:py:mod:`lsst.ap.pipe`
```

to get an actual link there. My `package-docs` build is failing right now (I think I'm out of date on some things), so I can't check.

Reply:
Oh is that the new way to do it? I thought that only worked for "normal" python modules and not our own, as the rst guide only has very generic examples. But I think you're right. The failure state is bold text in lieu of a link, which looks better than squiggle thingy at any rate.